extra-multimedia/feeluown/autobuild/defines

```diff
@@ -1,11 +1,12 @@
 PKGNAME=feeluown
 PKGSEC=sound
-PKGDEP="python-xlib quamash requests sqlalchemy xdg-utils mpv fuocore future"
-BUILDDEP="setuptools"
 PKGDES="A universal player for various Chinese music streaming services"
+PKGDEP="python-3 janus qasync pyqt5 pyopengl requests toml mpv \
+        feeluown-download feeluown-local feeluown-kuwo feeluown-netease feeluown-qqmusic feeluown-xiami"
+PKGBREAK="fuocore"
```

Comment:
You should actually break *and* replace an older fuocore version, defining the last version of fuocore available from the repository.

See the [Debian Policy Manual](https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-in-other-packages).